modules/Bio/EnsEMBL/Compara/PipeConfig/PairAligner_conf.pm

```diff
@@ -507,7 +507,7 @@ sub core_pipeline_analyses {
 	       -flow_into => {
 			       1 => [ 'update_max_alignment_length_after_net' ],
 			   },
- 	       -wait_for =>  [ 'alignment_nets', 'alignment_nets_himem', 'alignment_nets_hugemem', 'create_alignment_nets_jobs' ],    # Needed because of bi-directional netting: 2 jobs in create_alignment_nets_jobs can result in 1 job here
+ 	       -wait_for =>  [ 'alignment_nets', 'alignment_nets_himem', 'alignment_nets_hugemem', 'create_alignment_nets_jobs', 'filter_duplicates_net' ],    # Needed because of bi-directional netting: 2 jobs in create_alignment_nets_jobs can result in 1 job here
```

Comment:
This is creating a circular lock that worked for your test because `filter_duplicate_nets` didn't have any jobs when the job at `remove_inconsistencies_after_net` was ready (or at least that is my interpretation of what has happened).
Based on @muffato's comment on the JIRA ticket, I think one solution is to "clone" the analysis `remove_inconsistencies_after_net` and put if after `filter_duplicate_nets` under a new name like `remove_inconsistencies_after_filter`.